### DISCUSSION TITLE
Getting original request url onopen

### DISCUSSION THREAD

**Discussion by azertyalex** [uNetworking/uWebSockets#1442] [Q&A] [answered]
When a new websocket clients connect I pass url parameters in my request.  I can't find a way to access the original request uri, is there a way I can?

**Comment by ghost** [comment, marked as answer]
Look at example Upgrade. You get HTTP stuff in onupgrade

**Comment by azertyalex** [comment]
Alright thanks :)

I successfully moved my project from seasocks to uWebSockets now
